packages/prisma/src/prisma/schema.prisma

```diff
@@ -199,25 +261,117 @@ model Question {
   updatedAt DateTime @updatedAt
 }
 
-model Tag {
-  id   Int    @id @default(autoincrement())
-  name String
+// TODO: elements are only supported for the new PRACTICE_QUIZ type in the first iteration. add the other types here when working on them.
+enum ElementInstanceType {
+  PRACTICE_QUIZ
```

Comment:
Could we also already add LIVE_QUIZ, MICROLEARNING and GROUP_ACTIVITY as types here so that we need one migration step less later on? :)

Reply:
same as above, I would add it once the activity supports elements